### ISSUE TITLE
ev.source is null in Firefox extensions

### ISSUE THREAD

**Issue by twolfson** [defunctzombie/node-process#2]
tl;dr In Firefox browser extensions, `ev.source` is `null` for postMessages.

I am writing a [Pinboard.in](https://pinboard.in/) extension on [Crossrider](http://crossrider.com/) and trying to use [browserify](https://github.com/substack/node-browserify) to:
- Get some nice APIs for the extension
- Experimentation with browserify for extensions

I just caught a bug which shows `ev.source` is `null` inside of the extension. I will submit a PR some time in the future (I am currently have awake).

"Instrumented" `process.nextTicket`
![Selection_005](https://f.cloud.github.com/assets/902488/424677/12184bf2-ad92-11e2-8e30-0f566786c26f.png)

Proof that `ev.source` is `null`
![Selection_006](https://f.cloud.github.com/assets/902488/424679/1c7d5538-ad92-11e2-978a-30aea02bd9c7.png)
